tests/Cargo.toml

```diff
@@ -13,6 +13,7 @@ path = "src/test-ftdetect.rs"
 clap         = { version = "4.5.4", features = ["derive"] }
 ctrlc        = "3.4.4"
 fancy-regex  = "0.13.0"
+once_cell    = "1.19.0"
```

Comment:
This is in fact provided by rust itself now! See [the docs](https://docs.rs/once_cell/latest/once_cell/).

Reply:
lmao I thought I'd pulled up the standard library docs but ended up in the crate docs! I am a fool. This is good to go, no need to rely on nightly.